profiles/common/modules/features/ec_embedded_video/includes/ec_embedded_video.formatters.inc

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * @file
+ * File formatters for Platform needs.
+ */
+
+/**
+ * Implements hook_file_formatter_info_alter().
+ *
+ * It alters file_formatter_info to add the "Wrapper free formatters for the
+ * different Video-sharing services (YouTube, Vimeo, Dailymotion and
+ * AV portal).
+ */
+function ec_embedded_video_file_formatter_info_alter(&$info) {
```

Comment:
~~Maybe a stupid question, but why adding those stuff in an alter ? This code should live in `hook_file_formatter_info()` no ?~~

Ok I got it, you are making a duplicate of the existing ones and update them.
I'm still wondering if it's the best place to declare them. Those alter stuff are not cached.